### PR TITLE
Task 19 - Refactor homepage

### PR DESCRIPTION
**Pull request by vinhpeters**
Make home page and navbar fully responsive.

### CODE REVIEW

File: client/src/components/MuiFooter.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+    // Container,
```
Comment (hjkmines) [inline client/src/components/MuiFooter.js]: are you going to be using this in the future `Container`? If not, let's remove it.

File: client/src/components/MuiFooter.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    Box,
+    Button
+ } from '@mui/material';
+import  Link  from '@mui/material/Link';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+// import LinkTree from './LinkTree';
```
Comment (hjkmines) [inline client/src/components/MuiFooter.js]: if we are not using this, let's remove this as well.

**Review by hjkmines — CHANGES_REQUESTED** [review]

File: client/src/components/MuiFooter.js (hunk 1 above)
Comment (vinhpeters) [reply client/src/components/MuiFooter.js]: Removed old footer component.

File: client/src/components/MuiFooter.js (hunk 2 above)
Comment (vinhpeters) [reply client/src/components/MuiFooter.js]: Removed old footer component.

**Review by hjkmines — APPROVED** [review]
LGTM sir!

**Review by vmaineng — APPROVED** [review]
Great job y'all!

**Comment by jkyan1988** [comment]
LGTM

**Comment by jkyan1988** [comment]
LGTM

**Review by FunkyColdMedyna — APPROVED** [review]
LGTM

**Review by jkyan1988 — APPROVED** [review]
LGTM

**Pull request merged**